src/services/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@bcrs-shared-components/services",
-  "version": "1.0.35",
+  "version": "1.0.36",
   "publishConfig": {
     "access": "public"
   },
   "dependencies": {
-    "@bcrs-shared-components/enums": "^1.1.21",
+    "@bcrs-shared-components/enums": "^1.1.22",
```

Comment:
As above, revert these changes. Lerna will do them automatically in step 14:
https://github.com/bcgov/bcrs-shared-components/blob/main/README.md

Reply:
To be clear: leave the extra dependency here but use current version numbers. Lerna will update the version numbers.